setup.py

```diff
@@ -18,7 +18,7 @@
             'readit = manager.__init__:main',
         ],
     },
-    version="0.1",
+    version="1.0",
```

Comment:
If I remember correctly, we are going to release the `v0.1`.

Reply:
@bhavin192 
yes, `v0.1`
I will update it.